Fix Akamai duplicated & missing events issue (#34810)
demisto/content: Packs/Akamai_SIEM/ReleaseNotes/1_1_4.md

#### Integrations

##### Akamai WAF SIEM

> - Fixed an issue where **fetch-events** would sometimes miss / duplicate events due to discrepancy in the fetching mechanism.
> - Fixed an issue where fetched events will sometimes fail decoding certain fields.

Reviewer: ```suggestion
- Fixed an issue where the ***fetch-events*** command would sometimes miss or duplicate events.
- Fixed an issue where fetched events sometimes failed to decode certain fields.
```
